target/linux/ramips/image/mt7620.mk

```diff
@@ -488,6 +488,25 @@ define Device/hnet_c108
 endef
 TARGET_DEVICES += hnet_c108
 
+define Device/hootoo_tm05
+  SOC := mt7620n
+  IMAGE_SIZE := 7808k
+  DEVICE_VENDOR := HooToo
+  DEVICE_MODEL := TM05
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-i2c-ralink
+  LOADER_TYPE := bin
+  LOADER_FLASH_OFFS := 0xFD051000
```

Comment:
@adrianschmutzler , wouldn't be better to separate the introduction of `loader-okli*` recipes from the device support commit?
If it were a separate OKLI commit, then [the `AR71XX_FLASH_START` could be fixed](http://lists.openwrt.org/pipermail/openwrt-devel/2020-May/028951.html) to the correct `ramips` value: `0x10000000`, and this negative offset could be eliminated.

Reply:
@xabolcs I agree, same PR but different commit

@arrmo I can show you how to split the commits now, but I think it should wait as editing them separately is a pain

Reply:
I don't see an urgent need to separate this at the moment. I'd see that as a matter of taste.

Reply:
OK, will leave it alone then. Thanks!

Reply:
> Is it really possible to set AR71XX_FLASH_START for specific board?

@mpratt14 if it were called as `CONFIG_FLASH_START` like `CONFIG_FLASH_OFFS`, then it would be possible.
But, there is no need to be board specific: as I previously noted, this could be `0x10000000` for the whole `ramips` target as it's the same for `mt7620` and `mt7621` subtargets.